src/Pages/home/Components/Title.tsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import Box from "@material-ui/core/Box";
+import Button from "@material-ui/core/Button";
+import Typography from "@material-ui/core/Typography";
+import { createStyles, makeStyles } from "@material-ui/core/styles";
+import AddIcon from "../../../assets/add.png";
+
+//todo Style
+const useStyle = makeStyles(() =>
+  createStyles({
+    root: {
+      width: "100%",
+      marginTop: "16px",
+      padding: "32px",
+    },
+
+    //! Box
+    boxWelcome: {
+      width: "45%",
+      height: "93px",
+      padding: "27px",
+      borderRadius: "20px",
+      background:
+        "linear-gradient(94.59deg, #5028C6 9.91%, #FD85FF 81.34%, #F40000 111.1%)",
+      boxShadow: "0px 2px 4px rgba(0, 0, 0, 0.1)",
+    },
+    boxTotalPatients: {
+      width: "10%",
+      height: "93px",
+      padding: "27px",
+      marginLeft: "24px",
+      background: "#FFFFFF",
+      boxShadow: "0px 2px 4px rgba(0, 0, 0, 0.1)",
+      borderRadius: " 20px",
+    },
+    //! Text
+    textWelcome: {
+      color: "white",
+      width: "50%",
+    },
+    textTotalPatients: {
+      color: "#979797",
+      fontWeight: "normal",
+    },
+    textRegister: {
+      textTransform: "none",
+      color: "white",
+      fontWeight: "lighter",
+    },
+    //! Button
+    buttonRegister: {
+      backgroundColor: "#EF5DA8",
+      width: "25%",
+      height: "144px",
+      borderRadius: "24px",
+      marginLeft: "47px",
+      marginRight: "32px",
```

Comment:
Generally, we set either marginLeft or marginRight. Please remove one(As I saw, line 31 was set marginLeft so this will be too).